src/index.js

```diff
@@ -0,0 +1,5 @@
+import './style.css';
+
+// const mainContainer = document.querySelector(".main-container");
+
+// let toDo = [];
```

Comment:
- [ ] **Required:** Please remove all commented code. Pushing commented code is unprofessional and constitutes unclean code.